### PR TITLE
Call the is_authenticated method

### PR DESCRIPTION
**Pull request by andizer**
We called the `get_refresh_token` method, but this method became protected. There is a new method `is_authenticated`, because this was what we want to know instead of getting a refresh token that won't be used further in the code.

Be sure pull https://github.com/Yoast/api-libs/pull/25 is merged before this one.

### CODE REVIEW

**Comment by CarolineGeven** [comment]
CR 1 done :+1: ;)

**Comment by andizer** [comment]
@CarolineGeven Can you review the last changes?

File: admin/class-google-analytics.php [hunk 1]
```diff
@@ -201,11 +201,13 @@ protected function set_client() {
 			'application_name' => 'Google Analytics by Yoast',
 			'client_id'        => '346753076522-21smrc6aq0hq8oij8001s57dfoo8igf5.apps.googleusercontent.com',
 			'client_secret'    => '5oWaEGFgp-bSrY6vWBmdPfIF',
+			'redirect_uri'     => 'urn:ietf:wg:oauth:2.0:oob',
```
Comment (CarolineGeven) [inline admin/class-google-analytics.php]: Could you explain, with inline documentation, what happens here in `redirect_uri` to improve readability?

**Comment by CarolineGeven** [comment]
CR2 done

File: admin/class-google-analytics.php (hunk 1 above)
Comment (andizer) [inline admin/class-google-analytics.php]: Nopez, this is de value Google is giving. Don't know what this is meaning. Look likes everything is allowed.

Comment (CarolineGeven) [inline admin/class-google-analytics.php]: Seems like the explanation can be found here: https://developers.google.com/identity/protocols/OAuth2InstalledApp#choosingredirecturi
Please give a short explanation and add this URL in the documentation for readability and to make sure we know what's going on here.

**Pull request merged**